assets/charts/components/headlamp/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for headlamp.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: joaquimrocha/headlamp
```

Comment:
should we push it to `kinvolk/headlamp`?

Reply:
OK. Pushed to quay.io/kinvolk/headlamp